core/imageroot/var/lib/nethserver/node/update-core.d/30restart_redis

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2023 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+exec 1>&2
+set -e
+
+# Restart Redis service
+systemctl restart redis.service
```

Comment:
How this affects the UI? I fear that the api-server can't retrieve the status of the task running the core update.
Have you tested it?

Reply:
IIRC the "Update core" notification on the UI has always worked properly to this day. @DavidePrincipi have you tested the behavior of this PR on the UI? Are you implying that some fixes or improvements are needed?